### PR TITLE
fix: update user_expire_step to accept days instead of date

### PR DESCRIPTION
**Pull request by erfjab**
With this feature, you don't need to enter the exact date anymore, just by entering the desired day, it will record the date itself.

### CODE REVIEW

**Comment by ImMohammad20000** [comment]
You can use d|m|y to do this now
5d => 5 days
2m => 2 month
1y => 1 year

Its better than only use days for expire date

File: app/telegram/handlers/admin.py [hunk 1]
```diff
@@ -382,7 +382,7 @@ def edit_user_command(call: types.CallbackQuery):
     elif action == "expire":
         msg = bot.send_message(
             call.message.chat.id,
-            '⬆️ Enter Expire Date (YYYY-MM-DD)\nOr You Can Use Regex Symbol: ^[0-9]{1,3}(M|D) :\n⚠️ Send 0 for never expire.',
```
Comment (M03ED) [inline app/telegram/handlers/admin.py:385]: you could just use regex , no need for this change

**Comment by M03ED** [comment]
no need for this pr , this is already available

**Comment by erfjab** [comment]
Yes, that's right. I hadn't paid attention to him.

**Pull request closed without merging**